nx-libs.spec

```diff
@@ -1,4 +1,8 @@
 %global _hardened_build 1
+# prevent "ERROR: No build ID note found" errors
+%if 0%{rhel} < 7
+%undefine _missing_build_ids_terminate_build
```

Comment:
Why exactly does this lead to a failure anyway?

According to [the RHEL6 documentation](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Developer_Guide/compiling-build-id.html) all binaries built on EL6+ should have buildid information (probably through the compiler.)

So, why is that failing for you?